<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TInterface</class>
 <widget class="TInterface" name="TInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>13</y>
     <width>1025</width>
     <height>751</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayt">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <item>
     <widget class="QLabel" name="title">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Модель вычислительного центра</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>480</width>
          <height>355</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Управление</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <widget class="QLabel" name="taskIntLbl">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>211</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Интенсивность задач (в час):</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="taskIntSBox">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>20</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
        <widget class="QLabel" name="taskTimeLbl">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>231</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Ср. длительность задачи (мин): </string>
         </property>
        </widget>
        <widget class="QSpinBox" name="taskTimeSBox">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>60</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
        <widget class="QLabel" name="breakIntLbl">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>241</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Интенсивность сбоев ПК (в час):</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="breakIntSBox">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>100</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
        <widget class="QPushButton" name="startBtn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>170</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Запуск</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pauseBtn">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>170</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Пауза</string>
         </property>
        </widget>
        <widget class="QPushButton" name="stopBtn">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>170</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Стоп</string>
         </property>
        </widget>
        <widget class="QPushButton" name="setBtn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Установить</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="minimumSize">
         <size>
          <width>480</width>
          <height>355</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Статус</string>
        </property>
        <widget class="QWidget" name="gridLayoutWidget">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>19</y>
           <width>461</width>
           <height>331</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>PC1:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>PC4:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>PC2:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>PC5:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>PC3:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="status0">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>off</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="status1">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>off</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="status2">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>off</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="status3">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>off</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="status4">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>off</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="title">
         <string> Статистика</string>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>19</y>
           <width>951</width>
           <height>321</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QFrame" name="frame">
            <property name="minimumSize">
             <size>
              <width>460</width>
              <height>300</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="label">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>50</y>
               <width>131</width>
               <height>21</height>
              </rect>
             </property>
             <property name="text">
              <string>Задач:</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_2">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>80</y>
               <width>91</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>Выполнено:</string>
             </property>
            </widget>
            <widget class="QSpinBox" name="taskDoneSBox">
             <property name="geometry">
              <rect>
               <x>130</x>
               <y>80</y>
               <width>42</width>
               <height>22</height>
              </rect>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
            <widget class="QLabel" name="label_3">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>110</y>
               <width>101</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>Не принято:</string>
             </property>
            </widget>
            <widget class="QSpinBox" name="notRecTaskSBox">
             <property name="geometry">
              <rect>
               <x>130</x>
               <y>110</y>
               <width>42</width>
               <height>22</height>
              </rect>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
            <widget class="QLabel" name="label_4">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>140</y>
               <width>101</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>Прервано:</string>
             </property>
            </widget>
            <widget class="QSpinBox" name="canceledTaskSBox">
             <property name="geometry">
              <rect>
               <x>130</x>
               <y>140</y>
               <width>42</width>
               <height>22</height>
              </rect>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
            <widget class="QLabel" name="label_5">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>180</y>
               <width>251</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>Среднее время между сбоями ПК:</string>
             </property>
            </widget>
            <widget class="QDoubleSpinBox" name="avTimeSBox">
             <property name="geometry">
              <rect>
               <x>260</x>
               <y>180</y>
               <width>62</width>
               <height>22</height>
              </rect>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="maximum">
              <double>1000.000000000000000</double>
             </property>
            </widget>
            <widget class="QLabel" name="label_11">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>220</y>
               <width>231</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>СКО длительности интервалов:</string>
             </property>
            </widget>
            <widget class="QDoubleSpinBox" name="SDSBox">
             <property name="geometry">
              <rect>
               <x>250</x>
               <y>220</y>
               <width>62</width>
               <height>22</height>
              </rect>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="maximum">
              <double>1000.000000000000000</double>
             </property>
            </widget>
            <widget class="QLabel" name="label_12">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>20</y>
               <width>171</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>Часов смоделировано:</string>
             </property>
            </widget>
            <widget class="QSpinBox" name="cycleCount">
             <property name="geometry">
              <rect>
               <x>190</x>
               <y>20</y>
               <width>42</width>
               <height>22</height>
              </rect>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="statWgt">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TInterface</class>
   <extends>QWidget</extends>
   <header>tinterface.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
